### PR TITLE
add draft announcement for Slack

### PR DESCRIPTION
**Pull request by Bradamant3**
Word hard. Help please. MVP only, I've had this in mind for a while and we could do more but we need to ship something sooner rather than later. n.b. links not included yet but are intended to be added.

Doing it this way because:
- seems source should live somewhere public
- at root as text file bc initially all I can think of is manual copy/paste from git fork to announcement channel in the Slack. Automation would be good eventually.

### CODE REVIEW

**Comment by plaindocs** [comment]
I think you can use the built in `/remind #general "The Nice message" every month` although I'm not 100% sure about the last every `month`.

https://get.slack.help/hc/en-us/articles/208423427-Set-a-reminder

And we could probably find a better home for this in the org-guide as in include file than in the file root.

File: slack_announce.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
+
+We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our basic channels, and ponder our social rules. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal!
```
Comment (plaindocs) [inline slack_announce.txt]:
```suggestion
We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our social rules. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal!
```

**Review by plaindocs — APPROVED** [review]

**Comment by Bradamant3** [comment]
Ugh per Slack help and my own local tests, Slack does not support markdown or rst-style links. This is the best you can do. Alternative suggestions welcome!

Reminder: this is .txt bc the intent is to post this copy as a Slack message.

File: docs/include/slack.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
+
+We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our social rules. All this information is at https://www.writethedocs.org/slack/ -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal!
```
Comment (ericholscher) [inline docs/include/slack.txt:3]: Feels like this ends a bit oddly, perhaps:

```suggestion
We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our social rules. All this information is at https://www.writethedocs.org/slack/ -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal, so please keep them in mind while you're here! :tada:
```

**Review by ericholscher — APPROVED** [review]
> I think you can use the built in /remind #general "The Nice message" every month although I'm not 100% sure about the last every month.

This is a good idea. :+1:

File: docs/include/slack.txt (hunk 2 above)
Comment (plaindocs) [reply docs/include/slack.txt:3]: :+1: although punctuation good

**Pull request merged**